pages/common/clangd.md

```diff
@@ -2,7 +2,7 @@
 
 > Language server that provides IDE-like features to editors.
 > It should be used via an editor plugin rather than invoked directly.
-> More information: <https://manpages.ubuntu.com/manpages/focal/man1/clangd.1.html>.
+> More information: <https://manpages.ubuntu.com/manpages/noble/en/man1/clangd.1.html>.
```

Comment:
```suggestion
> More information: <https://manpages.ubuntu.com/manpages/man1/clangd.1.html>.
```

Reply:
@Managor why so? in the end you are redirected to `https://manpages.ubuntu.com/manpages/noble/en/man1/clangd.1.html`